docs/uml-class-diagrams/middleware01/test/MiddlewareGisProcessorImplementationTests/MiddlewareGisProcessorImplementationTests.puml

```diff
@@ -1,9 +1,11 @@
 @startuml MiddlewareGisProcessorImplementationTests
 
 package edu.oakland.test.middleware01 {
-    class MiddlewareGisProcessorImplementationTests {
-
-    }
+  class MiddlewareGisProcessorImplementationTests {
+    + void isGpsSignalStrengthNull()
+    + void checkValueGpsSignalStrength()
```

Comment:
Just updating the test plan document to match what was implemented.
```suggestion
    + void passSignalStrengthUc1()
    + void passSignalStrengthUc2()
```

Reply:
Isn't what you commented on for the the Processor? We had the Uc1 and Uc2 confusion in the Interface. @andrewdimmer